index.html

```diff
@@ -263,7 +292,7 @@ <h3 id="form-headline">
         <li class="footer-social-media-button"><a href="https://twitter.com/mperezbazan"><img src="./images/Twitter-dark.svg" alt="Twitter icon"/></a></li>
         <li class="footer-social-media-button"><a href="https://medium.com/@martin.perezbazan"><img src="./images/Medium-dark.svg" alt="Medium icon"/></a></li>
```

Comment:
- [ ] Great job making the requested changes but you forgot to make sure that your social media icons are displaying properly so please try to fix that 
<img width="312" alt="Screen Shot 2022-09-29 at 18 42 56" src="https://user-images.githubusercontent.com/61790587/193104851-98919d47-a2d7-4c0b-adff-823f04ca190f.png">

Reply:
@youmari Thanks for your feedback, I checked it on my PC, and my coding partner checked on his PC and it's working well, all the social media icons were in the place